2018-September-18/writingSamples/harmata_sample.tex

```diff
@@ -84,56 +84,76 @@
 is common, especially in temporal lobe seizures, but that prolonged
+% looks good
 postictal apnea is far less common\textsuperscript{4}. Similarly,
 post-stimulation breathing changes are not always observed in our work,
 as we have studied numerous (18) other patients who did not show
 pronounced, long-lasting hypoventilation after the end of amygdala
+% looks good
 stimulation. However, we did note that the patients who did have
 prolonged breathing alterations (P352, P384, and P413) all had seizure
+% Thus, it... may be better as two/three sentences
 foci involving the temporal lobe. Thus, it is possible that prolonged
 breathing disruption following stimulation or seizure could represent a
-biomarker specific to a subset of patients, and that abnormal
-connectivity of the mesial temporal lobe after years of uncontrolled
-seizures may be important for breathing to remain disrupted after the
-end of amygdala stimulation or seizure.
+biomarker specific to a subset of patients. One such biomarker could 
+be abnormal connectivity of the mesial temporal lobe after years of 
+uncontrolled seizures. Proper connectivity may be important for breathing
+to remain disrupted after the end of amygdala stimulation or seizure.
 
+%looks good
 However, it is possible that the prevalence of prolonged breathing
+% looks good
 disruption is under-recognized. For seizures observed in epilepsy
 monitoring units, a typical clinical approach is to intervene by
 immediately asking the patient follow-up questions and performing a
+% looks good
 neurological exam. This would disrupt breathing signal immediately
 following the seizure and potentially mask changes by causing the
+% looks good
 patient to voluntarily breathe in order to talk. Thus, it is possible
 that postictal apnea or irregular breathing is more common than
+% looks good
 anticipated. We were able to gather valuable postictal data for P413 by
 adapting the intervention protocol for a partial seizure by limiting
 disruption of natural breathing patterns. Use of this approach when
 possible could reveal more instances of postictal breathing disruption
+% looks good (maybe split into two sentences...)
 than currently detected, and may be critical for understanding SUDEP
 events that occur when the patient is unlikely to be prompted to talk
 (alone, at night, in bed, etc.).
 
+% looks good
 It is also possible that more patients may exhibit prolonged breathing
 disruption with amygdala stimulation, but were not detected due to
+% looks good
 variations in electrode placement. For the patients we describe in this
 report, site of amygdala stimulation was crucial in whether or not apnea
 occurred at all, and whether or not apnea persisted after stimulation
+% looks good
 ceased. Thus, it is possible that stimulation or seizure must occur in a
 specific location to cause acute and/or prolonged effects, even within
+% looks good
 the same patient. It is therefore possible that more patients with
```

Comment:
question: what is the relative specificity of a seizure foci versus an electrode? Is it more likely that a seizure would alter the activity of the necessary location of the amygdala?

Reply:
It's relationship to the writing should we interpret apnea caused by stimulus versus seizure.